lib/SPIRV/libSPIRV/SPIRVDecorate.h

```diff
@@ -448,6 +450,17 @@ class SPIRVDecorateMergeINTELAttr : public SPIRVDecorate {
   }
 };
 
+class SPIRVDecorateBankBitsINTELAttr : public SPIRVDecorate {
+public:
+  // Complete constructor for BankBitsINTEL decoration
+  SPIRVDecorateBankBitsINTELAttr(SPIRVEntry *TheTarget,
+                                 const std::vector<SPIRVWord> &TheBits)
+      : SPIRVDecorate(DecorationBankBitsINTEL, TheTarget) {
+    Literals = TheBits;
```

Comment:
Why not to move it even in creation block:
`
    : SPIRVDecorate(DecorationBankBitsINTEL, TheTarget),
      Literals(TheBits) {
      ....
`

Reply:
@vmaksimo Please apply comment from @mlychkov  and squash the commits into one. Thanks.

Reply:
It's not possible to access this field this way, because it's a field in a parent class. We can only initialize fields of the current class and call the constructors of parent classes, but they don't contain Literals vector initialization.